kmk/extensions/oled.py

```diff
@@ -0,0 +1,291 @@
+import busio
+import gc
+from supervisor import ticks_ms
+
+import adafruit_displayio_ssd1306
+import displayio
+import terminalio
+from adafruit_display_text import label
+
+from kmk.extensions import Extension
+from kmk.handlers.stock import passthrough as handler_passthrough
+from kmk.keys import make_key
+from kmk.kmktime import check_deadline
+
+
+class OledEntryType:
+    TXT = 0
+    IMG = 1
+
+
+class OledData:
+    def __init__(
+        self,
+        entries=None,
+    ):
+        if entries is not None:
+            self.data = entries
+
+    @staticmethod
+    def oled_text_entry(
```

Comment:
I don't get why this is a static method in a class that is basically just a list. Why not
```python
Oled(views = [
    OledTextEntry(...),
    OledImageEntry(...),
    ...], ...
)
```

Reply:
Again, something inherited, will fix that.